.gitignore

```diff
@@ -28,6 +28,7 @@ yarn-error.log*
 # local env files
 .env
 .env.local
+.env.development
```

Comment:
Please keep in mind that **.env.development** is a template file for environment variables and we want to keep this updated if we introduce any new properties.